app/src/hooks/featureLimiter/featureLimitTypes.ts

```diff
@@ -0,0 +1,21 @@
+import APP_CONSTANTS from "config/constants";
+
+export enum FeatureLimitType {
+  num_rules = "num_rules",
+  num_active_rules = "num_active_rules",
+}
+
+type FeatureLimits = Record<FeatureLimitType, number | boolean>;
+
+type PlanFeatureLimits = Record<string, FeatureLimits>;
+
+export const featureLimits: PlanFeatureLimits = {
+  [APP_CONSTANTS.PRICING.PLAN_NAMES.FREE]: {
+    [FeatureLimitType.num_rules]: 10,
+    [FeatureLimitType.num_active_rules]: 3,
+  },
+  [APP_CONSTANTS.PRICING.PLAN_NAMES.BASIC]: {
+    [FeatureLimitType.num_rules]: 25,
+    [FeatureLimitType.num_active_rules]: 10,
+  },
+};
```

Comment:
Lets only maintain types in this file. Can we move this to a new `constants.ts` file?

Reply:
Moved the types to `types.ts`